Services/Administration/OJS.Services.Administration.Business/Implementations/SubmissionsForProcessingBusinessService.cs

```diff
@@ -31,5 +48,31 @@ public async Task ResetAllProcessingSubmissions()
                 await this.submissionsForProcessingData.ResetProcessingStatusById(submissionForProcessingId);
             }
         }
+
+        public void EnqueueStaleSubmissions()
+        {
+            var submissionsForProcessing = this.submissionsForProcessingData
+                .GetAllPending()
+                .ToList()
+                .Where(sfp => Math.Abs(sfp!.CreatedOn.Subtract(DateTime.UtcNow).TotalMinutes) >= 1);
+
+            if (!submissionsForProcessing.Any())
+            {
+                return;
+            }
+
+            var submissions = this.submissionsData
+                .GetByIds(submissionsForProcessing
+                    .Select(sp => sp!.SubmissionId)
+                    .ToList())
+                .ToList();
+
+            submissions
+                .ForEachSequential((submission) =>
+                    this.submissionsCommonBusinessService
+                    .PublishSubmissionForProcessing(submission.Map<SubmissionServiceModel>()))
```

Comment:
Can you MapCollection directly to SubmissionServiceModel, instead of doing Map one by one from materialzed submissions?

Reply:
Done